starter_code/src/movies.js

```diff
@@ -3,10 +3,27 @@
 
 
 // Get the average of all rates with 2 decimals 
+function ratesAverage(array) {
+  var sumOfRatings=array.reduce(function(acc, item){
+    var rateNumber=item.rate*1;    
+    return acc+rateNumber;
+  },0)
 
+  return parseFloat((sumOfRatings/array.length).toFixed(2));
+}
+console.log(ratesAverage(movies));
 
-// Get the average of Drama Movies
 
+// Get the average of Drama Movies
+function dramaMovieRate(array) {
+  var dramaMovie=array.filter(function(item){
+    return (item.genre.indexOf('Drama')===1);
```

Comment:
Care! 
This filter will return the elements that have the 'Drama' in the second position of `genre`. So will not get the elements that have genre like `['Drama']`.
One way to approach this, can be:
```
return (item.genre.indexOf('Drama') !== -1);
```
With this will take all the elements that have `Drama` on the array `genre`